core/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsFeedLoader.java

```diff
@@ -100,6 +100,9 @@ public GtfsFeedContainer loadAndValidate(
     Map<String, GtfsTableDescriptor<?>> remainingDescriptors =
         (Map<String, GtfsTableDescriptor<?>>) tableDescriptors.clone();
     for (String filename : gtfsInput.getFilenames()) {
+      if (filename.contains("translations")) {
+        AnyTableLoader.setHasTranslations(true);
+      }
       GtfsTableDescriptor<?> tableDescriptor = remainingDescriptors.remove(filename.toLowerCase());
```

Comment:
I noticed here that we set the filename to lowercase. Is it because a filename with uppercase letters is allowed?
If it is, I would also make sure we do a lowercase on the filename when checking for the presence of the `translation` string.